src/components/layout/CoinRateForm/index.tsx

```diff
@@ -7,116 +7,117 @@ import { Coin } from '../../../constants/coins';
+					className={styles.input}
+					onChange={onChange}
+					containerClassName={styles.container_input}
+					placeholder={placeholder}
+					right={<div className={styles.right}>{`${coin}x/mo.`}</div>}
+					type="number"
+				/>
+			</div>
+			<div className={styles.buttons}>
+				<div className={styles.start_wrap}>
+					<ButtonNew
+						loaderColor="white"
+						color="primary"
+						onClick={onClickStart}
+						className={styles.start}
+						disabled={
+							isReadOnly ||
+							isLoading ||
+							!value ||
+							((Math.floor(((parseFloat(value) / 2592000) * 1e18) / shareScaler) * shareScaler) / 1e18) *
+								2592000 ===
+								0
+						}
+						isLoading={isLoading}
+						data-tip
+						data-for={`depositTooltip-${indexVal}`}
+					>
+						{t('Start')}/{t('Edit')}
+					</ButtonNew>
+				</div>
+				<div className={styles.stop_wrap}>
+					{parseFloat(personalFlow) > 0 && (
+						<ButtonNew
+							loaderColor="#363B55"
+							color="secondary"
+							onClick={onClickStop}
+							className={styles.stop}
+							disabled={isReadOnly || isLoading}
+							isLoading={isLoading}
+						>
+							{t('Stop')}
+						</ButtonNew>
+					)}
+				</div>
+				<div style={{ flexBasis: '100%', height: '0' }}> </div>
 
-        {parseFloat(value) > 0 ?
-          (
-            <ReactTooltip
-              id={`depositTooltip-${indexVal}`}
-              place="right"
-              effect="solid"
-              multiline
-              className={styles.depositTooltip}
-            >
-              {value && coin && (
-              <span
-                className={styles.depositTooltip_span}
-              >
-                The amount per month will be rounded off to
-                <span style={{ fontWeight: 700 }}>
-                  {` ${(((Math.floor(((parseFloat(value) / 2592000) * 1e18) / shareScaler) * shareScaler) / 1e18) * 2592000).toFixed(6)} ${coin} `}
-                </span>
-                so the contracts can evenly divide it
-                and it will take a security deposit of
-                <span style={{ fontWeight: 700 }}>
-                  {` ${(parseFloat(value) / 180.0).toFixed(6)} ${coin} `}
-                </span>
-                from your balance.
-                The Deposit will be refunded in full when you close the stream or lost if
-                your balance hits zero with the stream still open.
-
-              </span>
-              )}
-            </ReactTooltip>
-          )
-          : null}
-        <div />
-      </div>
-    </div>
-  );
-};
\ No newline at end of file
+				{parseFloat(value) > 0 ? (
+					<ReactTooltip
+						id={`depositTooltip-${indexVal}`}
+						place="right"
+						effect="solid"
+						multiline
+						className={styles.depositTooltip}
+					>
+						{value && coin && (
+							<span className={styles.depositTooltip_span}>
+								The amount per month will be rounded off to
+								<span style={{ fontWeight: 700 }}>
+									{` ${(
```

Comment:
Can we bind this logic to a variable and use that in <span>? Again decoupling logic from UI components